include/exadg/incompressible_navier_stokes/user_interface/parameters.cpp

```diff
@@ -556,23 +556,13 @@ Parameters::check(dealii::ConditionalOStream const & pcout) const
   }
 
   // VARIABLE VISCOSITY MODELS
-  if(viscosity_is_variable())
+  if(viscosity_is_variable() &&
+     temporal_discretization == TemporalDiscretization::BDFDualSplittingScheme)
   {
-    if(temporal_discretization == TemporalDiscretization::BDFDualSplittingScheme)
-    {
-      AssertThrow(
-        treatment_of_variable_viscosity == TreatmentOfVariableViscosity::Explicit,
-        dealii::ExcMessage(
-          "An implicit treatment of the variable viscosity field (rendering the viscous step of the dual splitting scheme nonlinear regarding the unknown velocity field) is currently not implemented for the dual splitting scheme."));
-    }
-
-    if(treatment_of_variable_viscosity == TreatmentOfVariableViscosity::Implicit)
-    {
-      AssertThrow(
-        implicit_convective_problem(),
-        dealii::ExcMessage(
-          "The current implementation only calls a Newton solver, if we have an implicit convective problem. Treat nonlinear convective and viscous terms alike."));
-    }
+    AssertThrow(
+      treatment_of_variable_viscosity == TreatmentOfVariableViscosity::Explicit,
+      dealii::ExcMessage(
+        "An implicit treatment of the variable viscosity field (rendering the viscous step of the dual splitting scheme nonlinear regarding the unknown velocity field) is currently not implemented for the dual splitting scheme."));
```

Comment:
As a minor comment: Maybe you could split up this message into smaller pieces (as we do in other places of the code) for the purpose of indentation.